### PR TITLE
Some renames for datasave.c

### PR DESCRIPTION
**Pull request by giuse94**

### CODE REVIEW

File: src/Menu/datasave.c [hunk 1]
```diff
@@ -95,34 +95,49 @@ extern char aResultX[];
...
-    0x88A4635,  0x217414EF, 0x363929D6, 0x49035844, 0x2A4139A2, 0x1E8316E0, 0x10013444, 0xE021020,  0x12221234,
-    0xA010031,  0x12224,    0x32,       0x10001224, 0x200030,   0x10022324, 0x410332,   0x32233434, 0x210333,
-    0x44444444, 0x110233,   0x44444444, 0xA010133,  0x44444434, 0xCA10133,  0x34444432, 0xECBA023,  0x34433443,
-    0xEEDD013,  0x33421211, 0xEEED102,  0x23333222, 0xEEEC101,  0x12334310, 0xEED0100,  0x1233444,  0xEEA1002,
-    0,          0,          0x14A90425, 0x35B1252D, 0x88A4635,  0x217414EF, 0x363929D6, 0x44693427, 0x60CD508B,
-    0x64896CEF, 0x11001123, 0xED00210,  0x10000003, 0xEB01100,  0x11201013, 0xB200210,  0x12212124, 0xB020221,
-    0x22322234, 0xC211123,  0x44322144, 0xDB10123,  0x33321234, 0xEC01012,  0x23211023, 0xEEB0112,  0x22122102,
-    0xEED1121,  0x21222332, 0xEED2122,  0x10121100, 0xEEC0122,  0x10123433, 0xEEA0021,  0x12222100, 0xEC00021,
-    0x1221111,  0xEA00102,  0x22222333, 0xB000301,  0,          0
+int clutsAndIcons_8009E774[] = {
```
Comment (KieronJ) [inline src/Menu/datasave.c:98]: It's probably int[][40]

Comment (giuse94) [reply src/Menu/datasave.c]: Done!

File: src/Menu/menuman.h [hunk 2]
```diff
@@ -146,7 +146,7 @@ typedef struct DATA_INFO
 {
   char                        field_0[2];
   char                        field_2;
-  char                        field_3;      // block count
+  char                        blocksCount;
```
Comment (KieronJ) [inline src/Menu/menuman.h:149]: One last nitpick, could you use snake case for these names?

Comment (giuse94) [reply src/Menu/menuman.h:149]: No problem, but I can't do it before tomorrow.

**Pull request merged**